modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -36,10 +53,15 @@ def beneficiary
         render json: Mobile::V0::LettersBeneficiarySerializer.new(@current_user, response)
       end
 
-      # returns a pdf of the requested letter type given the user has that letter type available
+      # returns a pdf or json representation of the requested letter type given the user has that letter type available
       def download
+        if params[:format] == 'json'
```

Comment:
Is there any concern that the front end is accidentally requesting format `json` even though the `mobile_lighthouse_letters` flipper may not be enabled for this user? The logic here ignores the flipper altogether and just assumes the user has lighthouse flipped. May not be a big issue, but just waned to confirm

Reply:
Yeah, I'm ignoring the flipper flag because on the assumption that the mobile app is not sending a format param because this is entirely new functionality. If they are... well, I'd like to know why they are. I can spot check requests in datadog, but it feels like a safe bet that they aren't.